spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
   context 'autocomplete results' do
     it 'returns collection of matches' do
       7.times { create(:institution, :contains_harv) }
-      get :autocomplete, term: 'harv'
+      get :autocomplete, term: 'harv', version: 1
```

Comment:
I think in routes.rb the version should also maybe have a default / fallback for when it is not provided?